backend/src/services/buildingService.js

```diff
@@ -6,86 +6,96 @@ const errorMessages = {
 };
 
-const validateKingdomId = async (id) => {
-  const kingdomBase = await kingdomRepo.getKingdomBaseData({'kingdom_id' : id});
-  if ( kingdomBase.length === 0 ) throw new Error(errorMessages.invalidId);
+const validateKingdomId = async id => {
+  const kingdomBase = await kingdomRepo.getKingdomBaseData({ kingdom_id: id });
+  if (kingdomBase.length === 0) throw new Error(errorMessages.invalidId);
 };
 
-const getResourceChanges = async (id,type) => {
+const getResourceChanges = async (id, type) => {
   const resources = await resourceService.getResource(id);
   const gold = resources.resources.filter(e => e.type === 'gold')[0].amount;
-  if ( gold < buildingStats[type].cost ) throw new Error(errorMessages.notEnoughGold);
+  if (gold < buildingStats[type].cost)
+    throw new Error(errorMessages.notEnoughGold);
   return {
     cost: buildingStats[type].cost,
     genAdd: buildingStats[type].gen,
-    genType: buildingStats[type].genType
+    genType: buildingStats[type].genType,
   };
 };
 
-const add = async (input) => {
-    validateType(input.type);
-    await validateKingdomId(input.kingdomId);
-    const resourcesData = await getResourceChanges({kingdomID:input.kingdomId},input.type);
-    const buildingDataInput = createBuildingData(input);
-    const buildingId = await buildingRepo.add( buildingDataInput );
-    switch (input.type) {
-      case 'academy':
-        await resourceService.spendGold(input.kingdomId,resourcesData.cost);
-        break;
-      case 'farm':
-        await resourceService.spendGold(input.kingdomId,resourcesData.cost);
-        await resourceService.updateFoodGeneration(input.kingdomId,resourcesData.genAdd);
-        break;
-      case 'mine':
-        await resourceService.spendGold(input.kingdomId,resourcesData.cost);
-        await resourceService.updateGoldGeneration(input.kingdomId,resourcesData.genAdd);
-        break;
-      default:
-        break;
-    }
-    return formatData(buildingId,buildingDataInput);
+const add = async input => {
+  validateType(input.type);
+  await validateKingdomId(input.kingdomId);
+  const resourcesData = await getResourceChanges(
+    { kingdomID: input.kingdomId },
+    input.type
+  );
+  const buildingDataInput = createBuildingData(input);
+  const buildingId = await buildingRepo.add(buildingDataInput);
+  switch (input.type) {
+    case 'academy':
+      await resourceService.spendGold(input.kingdomId, resourcesData.cost);
+      break;
+    case 'farm':
+      await resourceService.spendGold(input.kingdomId, resourcesData.cost);
+      await resourceService.updateFoodGeneration(
+        input.kingdomId,
+        resourcesData.genAdd
+      );
+      break;
+    case 'mine':
+      await resourceService.spendGold(input.kingdomId, resourcesData.cost);
+      await resourceService.updateGoldGeneration(
+        input.kingdomId,
+        resourcesData.genAdd
+      );
+      break;
+    default:
+      break;
+  }
+  return formatData(buildingId, buildingDataInput);
```

Comment:
@kkeszthe these changes will spark real joy in @gulyaasjaanos eyes, because they will cause like the whole file to be conflicting :D

Reply:
@adamgyulavari do you expect action from me here or is it just a warning that I should not commit such changes in the future?

Reply:
the latter :D